docs/api-reference/core/model.md

```diff
@@ -326,12 +326,12 @@ How many instances will be rendered
 
 ### setGeometry() : Model
 
-Get model's `Geometry` instance
+Use a `Geometry` instance to define attribute buffers
```

Comment:
Worth philosophizing over what this method does?

Does it really set (replace) any geometry?

Maybe `setAttributesFromGeometry` would have been more suggestive?

A geometry is `{mode, attributes, instances}` or just `{mode, attributes}`

It also copies `mode`?